cmd/nodecmd/create_aws.go

```diff
@@ -97,8 +98,15 @@ func getAWSCloudConfig(awsProfile string) (map[string]*awsAPI.AwsCloud, map[stri
 	ec2SvcMap := map[string]*awsAPI.AwsCloud{}
 	amiMap := map[string]string{}
 	numNodesMap := map[string]int{}
+	// verify regions are valid
+	if validRegions, err := checkRegions(maps.Keys(finalRegions)); err != nil {
+		return nil, nil, nil, err
+	} else if !validRegions {
+		return nil, nil, nil, fmt.Errorf("invalid regions provided for %s", constants.AWSCloudService)
```

Comment:
can you point out to the user the given invalid regions?

Reply:
fixed

Reply:
I mean, to only print the invalid regions, not all the provided. I mean for checkRegions to not return a boolean, but the list of invalid regions

Reply:
fixed